swift/extractor/mangler/SwiftMangler.cpp

```diff
@@ -353,9 +353,10 @@ SwiftMangledName SwiftMangler::visitOpaqueTypeArchetypeType(
 }
 
 SwiftMangledName SwiftMangler::visitOpenedArchetypeType(const swift::OpenedArchetypeType* type) {
-  llvm::SmallVector<char> uuid;
-  type->getOpenedExistentialID().toString(uuid);
-  return visitArchetypeType(type) << std::string_view(uuid.data(), uuid.size());
+  // llvm::SmallVector<char> uuid;
+  // type->getOpenedExistentialID().toString(uuid);  // <- doesn't compile any more
+  // return visitArchetypeType(type) << std::string_view(uuid.data(), uuid.size());
+  return visitArchetypeType(type);
```

Comment:
Looking at https://github.com/swiftlang/swift/pull/76238 I see the following change:
```patch
-      printFieldQuoted(T->getOpenedExistentialID(), "opened_existential_id");
+      auto *env = T->getGenericEnvironment();
+      printFieldQuoted(env->getOpenedExistentialUUID(), "opened_existential_id");
```
where `T` is of type `OpenedArchetypeType`.